src/js/components/modals/CliInstallModal.js

```diff
@@ -65,14 +65,9 @@ class CliInstallModal extends React.Component {
     const clusterUrl = `${protocol}://${hostname}${port}`;
     const { selectedOS } = this.state;
 
-    let version = MetadataStore.parsedVersion;
-    // Prepend 'dcos-' to any version other than latest
-    if (version !== "latest") {
-      version = `dcos-${version}`;
-    }
     const downloadUrl = `https://downloads.dcos.io/binaries/cli/${
       osTypes[selectedOS]
-    }/x86-64/${version}/dcos`;
+    }/x86-64/latest/dcos`;
```

Comment:
@GeorgiSTodorov : I'm just catching-up with this change, can we please hardcode 1.13 here? That was the original intention when we opened https://github.com/dcos/dcos-ui/pull/3535

Reply:
@bamarni please help me understand why would we want having `1.13` hardcoded in `master` and when we should actually switch to always show `latest`?

Reply:
At the moment `latest` is the latest stable CLI (which is the one released for DC/OS 1.12), while `dcos-1.13` contains the DC/OS 1.13 CLI (which is currently a dev build, we're planning to push a stable version there at the time DC/OS 1.13 becomes beta).

It is important during the alpha / beta timeframe that customers use the 1.13 CLI, so they can get all the 1.13 specific features. Thus I'd rather have the `dcos-1.13` URL there.

For DC/OS 1.14 we will consider sticking to `latest` instead of `dcos-1.14`, assuming we can figure out how to handle the alpha/beta concern.

Reply:
@bamarni , I used this comment https://jira.mesosphere.com/browse/DCOS-16401?focusedCommentId=220189&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-220189 as guidance while working on this. Is the information in it no longer relevant?

Reply:
Indeed I was myself confused and didn't think about DC/OS beta :D

It is safer to stick to dcos-1.13 for now and we'll figure-out how to handle CLI URLs for beta DC/OS releases in the future, hopefully for 1.14.

Reply:
Since we're going to release 1.13 rather soon how critical is this?

Reply:
It is not highly critical but that'd be nice to change it. Is the UI release already frozen for 1.13?